packages/swapper/src/utils.ts

```diff
@@ -318,3 +325,37 @@ export const isToken = (assetId: AssetId) => {
 }
 export const isExecutableTradeStep = (step: TradeQuoteStep): step is ExecutableTradeStep =>
   step.accountNumber !== undefined
+
+export const checkSolanaSwapStatus = async ({
+  txHash,
+  chainId,
+  accountId,
+  assertGetSolanaChainAdapter,
+}: {
+  txHash: string
+  accountId: AccountId | undefined
+  chainId: ChainId
```

Comment:
suggestion: don't parametrize me. `chainId` makes sense for `checkEvmSwapStatus` since we're dealing with different EVM chains, but this method is called `checkSolanaSwapStatus`, not `checkSvmSwapStatus` so the `chainId` is guaranteed to always be solanaChainId